cmd/worker/main.go

```diff
@@ -196,15 +196,17 @@ func main() {
 		}
 		global.SetDB(db)
 
+		klasters := clusteradapter.NewClusters(db)
 		clusterManager := cluster.NewManager(
-			clusteradapter.NewClusters(db),
+			klasters,
```

Comment:
Is this intentional?

Reply:
`clusters` was already in use and naming is hard

Reply:
How about not putting it in a variable then?

Reply:
Because the value is used in more than one place?

Reply:
what about clusteradapter.Store interface extend the clusteradapter.Clusters interface? Or adding a method Clusters() to it which exposes the internal `clusters` field.

Reply:
The `Store` implementation (`struct`!) could be modified to expose the `Clusters`' methods. This way, a `Store` instance could be used in place of `Clusters` instances. Eventually, `Clusters` methods should be "merged" into `Store`, since they do mostly the same things.

Reply:
> what about clusteradapter.Store interface extend the clusteradapter.Clusters interface? Or adding a method Clusters() to it which exposes the internal clusters field.

That would defeat the whole purpose of this interface. 😕

Reply:
> Because the value is used in more than one place?

This is a thin layer on top of the database. I would rather create two instances and refactor later than use half hungarian, half english variable names.